Automattic/wp-calypso
# Add submission success page

#### Changes proposed in this Pull Request

* Added a new submission success page on `http://calypso.localhost:3000/marketplace/submission-success` route.
* Reused MasterbarStyled from the `http://calypso.localhost:3000/marketplace/thank-you/woocommerce-subscriptions/politicaemvideira.com` screen
#### Testing instructions



* Open the page http://calypso.localhost:3000/marketplace/thank-you/woocommerce-subscriptions/politicaemvideira.com and make sure the back button still works as expected
* Design should match the Figma file: SJ6I0rUUtHNdHMF2KMMJig-fi-311%3A5435
* Open the page http://calypso.localhost:3000/marketplace/submission-success and you should see this screen:

<img width="987" alt="image" src="https://user-images.githubusercontent.com/1044309/168652598-2dde3182-b4e8-4ba0-8960-d5cbfbee4d60.png">

* It should work in all supported browsers
* It should work on mobile



Related to https://github.com/Automattic/wp-calypso/issues/63554
